README.markdown

```diff
@@ -76,13 +76,24 @@ You will need:
    export GOOGLE_APP_ID="Google Project Id"
    ```
 
-   **Obtaining Google Credentials:** 
+   **Obtaining Google Credentials:**
+
        * Sign in to Google Developer Console
```

Comment:
Is this the URL, should we add it to this readme? https://console.developers.google.com/apis